test/integration/builder/selects.js

```diff
@@ -735,6 +735,51 @@ module.exports = function(knex) {
       });
     });
 
```

Comment:
Looks like there is no real integration tests at all checking that this kind of queries can actually be ran by sql drivers used by knex. So there should be some test, which also sends query until the database. Another thing that I want to see tested is putting ? in the comment string just to verify that those will not mess up parameter bindings.

Reply:
Good concerns @elhigu ! Would you agree these were addressed in the tests just under? (lines 738~759)